### DISCUSSION TITLE
Manage Inspection Targets with a Whitelist

### DISCUSSION THREAD

**Discussion by sundaycrafts** [JetBrains/Qodana#229] [Q&A]
Hello there,

I would like to manage inspection targets using a whitelist (not using `exclude:`), but I couldn't figure out how to do it from the official help page.
Could someone please help me?

**Comment by sundaycrafts** [comment]
bump

**Reply by tiulpin** [reply]
Thanks a lot for bumping! (really, because I missed the notification from the discussion on my mail, and now I've seen it)

**Comment by tiulpin** [comment]
@sundaycrafts most likely, what you are looking for is the "empty profile" https://www.jetbrains.com/help/qodana/qodana-yaml.html#include-example – you can do it via YAML profiles or `qodana.yaml` by specifying
```yaml
profile: empty
include: ...
```
